src/index.html

```diff
@@ -17,16 +17,29 @@
     />
```

Comment:
Consider adding a link to the 'fonts.css' file here to ensure that the custom fonts defined in 'src/fonts.css' are applied to the HTML document. For example: `<link rel="stylesheet" href="fonts.css" />`.